### ISSUE TITLE
Make this available in PyPI

### ISSUE THREAD

**Issue by gautamkrishnar** [jekyc/wig#34]
Please make this package available in http://pypi.python.org/ so that users can install it using:
```sh
pip install wig
```

I can help you out with the packaging and automatic deployment, if you are interested...

**Comment by jayvdb** [comment]
Yes please.  https://pypi.org/project/wig/ is available, so if you dont claim it, it might be gone when you try to do a release.
